Adds SolGov shuttle for players to play on

Bam40002 opened the pull request:
## What does it add?

Adds a playable SolGov shuttle.

Im planning to make it choose between map files so not every round the shuttle spawns in is the same map and the layout of the shuttle is different. (broken english)

## Why is it good for the game?

Could lead to fun interactions with the intrepid.

## Authorship

Me (based a bit on egors OBS shuttle with the code)

## Changelog

:cl: Bam4000
rscadd - Added offship solgov shuttle role
/:cl:

Bam40002 committed:
Adds the sol shuttle

-Adds a sol shuttle

Shuttle things

-Fixes airlock
-Random names!

(Still broken)

Varying Variable Troubles

-Changed the format to TGM
-Almost fixed the Airlock

Stupid Quotes

-added a " to a thing

Mapping Fixes

-Fixed some duplicate things on top of eachother.
-Removed some atmospherics pipes going into nowhere.

Mapping fixed.

-This map has issues and I fixed a bit

Duplicate Crap

-Catwalks on catwalks removed

Bam40002 commented:
For some reason it broke completely. It worked before a few months ago. 
Something must have changed. Boowomp

EgorDinamit commented on maps/away/sol_shuttle/sol_shuttle.dm:
```diff
@@ -0,0 +1,102 @@
+	)
+
+/obj/effect/overmap/visitable/ship/landable/sol_shuttle
+	name = "SolGov Cruiser"
```
Might want to give it a proper ship name, instead of simply saying it's "SolGov".
You know, the prefix, similar to "SEV" and "TGLV".

EgorDinamit commented on maps/away/sol_shuttle/sol_shuttle.dm:
```diff
@@ -0,0 +1,102 @@
+
+/obj/effect/overmap/visitable/ship/landable/sol_shuttle
+	name = "SolGov Cruiser"
+	desc = "Sensor array detects a small SolGov Cruiser."
```
Same here, but it could also use a bit more text to describe some details, maybe?

EgorDinamit commented on maps/away/sol_shuttle/sol_shuttle.dm:
```diff
@@ -0,0 +1,102 @@
+	vessel_size = SHIP_SIZE_SMALL
+
+/obj/effect/overmap/visitable/ship/landable/sol_shuttle/New()
+	name = "SEV [pick("Quantum", "Enterprise", "Event Horizon", "Notch","Discovery","Vagrant")]"
```
I'd imagine "SEV" doesn't exactly fit there, no? "SEV" stands for "Sol Exploratory Vessel", if I am not mistaken.

Bam40002 replied:
I will change it to SFV meaning Sol Fleet Vessel.

EgorDinamit commented on maps/away/sol_shuttle/sol_shuttle.dm:
```diff
@@ -0,0 +1,102 @@
+	name = "SEV [pick("Quantum", "Enterprise", "Event Horizon", "Notch","Discovery","Vagrant")]"
+	for(var/area/sol_shuttle/A)
+		A.name = "\improper [name] - [A.name]"
+		GLOB.using_map.area_purity_test_exempt_areas += A.type
```
Is it needed..?

Bam40002 replied:
Randomly picking a name? I think so.

EgorDinamit reviewed (commented):
The jobs would certainly need a rewrite or general improvements here and there.

EgorDinamit commented on maps/away/sol_shuttle/sol_shuttle_radio.dm:
```diff
@@ -0,0 +1,31 @@
+	preset_name = "SEV Quantum"
+
+/obj/machinery/telecomms/server/map_preset/sol_shuttle
+	preset_name = "SEV Quantum"
```
Might want to give it some generic channel name instead? Let alone, ship might not be even called that when it spawns.

Bam40002 replied:
On it! The SEV Quantum was the placeholder name.

Bam40002 committed:
Did things and added stuff.

-Fixed the shuttle, it works now
-Made a few mapping changes
-Added placeholder map for the second shuttle
-Changed job descriptions a bit

Bam40002 committed:
A lot of things happening

-Added new job the VIP (might need to remove one)
-Mapped a bit of the second shuttle
-Tweaked the first shuttle a bit
-Made it so when undocked the shuttle still has gravity

Bam40002 committed:
Dont have multiple projects at a time

Bam40002 commented:
DRIVING PLAYERS APART IS AN EVIL GAMEPLAY DECISION. ALL PLAYERS SHOULD STAY TOEGETHER HOKY FUCK
